src/languages/scala.js

```diff
@@ -137,6 +137,25 @@ export default function(hljs) {
     }
   };
 
+  const IMPORT_EXPORT = {
+    beginKeywords: 'import export',
+    end: /[;\n]/,
+    excludeEnd: true,
+    contains: [
+      { beginKeywords: 'as' },
+      {
+        begin: '{',
+        end: '}',
+        excludeBegin: true,
+        excludeEnd: true,
```

Comment:
Since there is no active scope here you shouldn't need these... the `{}` aren't part of any named highlighted region... unless I'm missing something?